app/views/shared/_collection_tabs.html.slim

```diff
@@ -34,18 +34,20 @@
       :name => t('.settings'),
       :selected => 5,
       :path => "#{edit_collection_path(@collection.owner, @collection)}",
-    },{\
-        :name => t('.export'),
-        :selected => 6,
-        :path => "#{collection_export_path(@collection.owner, @collection)}",
-     }]
+    }]
      
   -else
     -tabs += [{\
     :name => t('.settings'),
     :selected => 5,
     :path => "#{collection_settings_path(@collection.owner, @collection)}",
     }]
+
+  -tabs += [{\
```

Comment:
I'm not sure if the problem is here or elsewhere, but I am not seeing an "export" tab on a document set.

Reply:
I just tested this locally on CWRGM, and see the Export tab fine if I'm a project owner.